Source/Applications/MiMD/wwwroot/Scripts/TSX/MiMD/Configuration/ConfigurationFileRules.tsx

```diff
@@ -334,8 +334,8 @@ const ConfigurationFileRules = () => {
                                     { Value: 'IN', Label: 'IN' },
                                     { Value: '=', Label: '=' }
                                     ]} />
-                            <ConfigRuleValueTableField Record={currentRule} Edit={false} updateRule={(rule) => setCurrentRule(rule)} Label={'Value'} />
-                            <Select<MiMD.IConfigRules> Record={currentRule} Field={'AdditionalFieldID'} Disabled={false} Label={'Additional Field'} Setter={(rule) => setCurrentRule(rule)}
+                            <ConfigRuleValueTableField Record={currentRule} Edit={false} Help={'See the Help :question_mark: at the bottom of this window for dynamic expression examples.'} updateRule={(rule) => setCurrentRule(rule)} Label={'Value'}/>
```

Comment:
```suggestion
                            <ConfigRuleValueTableField Record={currentRule} Edit={false} Help={'See the Help ? at the bottom of this window for dynamic expression examples.'} updateRule={(rule) => setCurrentRule(rule)} Label={'Value'}/>
```

I think this needs a reword in general. Doesn't help have onclick for more information @prestoncraw or am I misremembering that?